### PR TITLE
feat home page

### PR DESCRIPTION
**Pull request by AmishMishra11**
added home page for the quiz app

https://user-images.githubusercontent.com/76205249/154209780-26a0798b-1791-447e-9e90-8fc708ef864c.mp4

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
+/* @import url(/stylesheet/default.css); */
```
Comment (amarphule) [inline styles.css]: remove comment

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  height: auto;
+}
+
+@media (max-width: 600px) {
+  .container-main {
+    height: 45vh;
+  }
+}
+
+@media (max-width: 400px) {
```
Comment (amarphule) [inline styles.css:86]: use `rem` instead `px`

File: stylesheet/default.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  --gap-largest: 2rem 3.5rem;
+  --gap-large: 1.5rem 2.5rem;
+  --gap-medium: 1rem 1.5rem;
+  --gap-small: 0.5rem 1rem;
+  --gap-smallest: 0.25rem 0.5rem;
+}
+
+/* border radius */
+
+:root {
```
Comment (amarphule) [inline stylesheet/default.css]: why multiple `:root` you can create a single root and use variables inside it

**Review by amarphule — APPROVED** [review]
Given some suggestions and please change text color as it it not visible
ami

File: styles.css (hunk 1 above)
Comment (AmishMishra11) [reply styles.css]: ok

File: stylesheet/default.css (hunk 3 above)
Comment (AmishMishra11) [reply stylesheet/default.css]: yes i will do that

File: styles.css (hunk 2 above)
Comment (AmishMishra11) [reply styles.css:86]: It's for media query so it is for a screen size of 400px and below, rem would not me effective here I think

File: index.html [hunk 4]
```diff
@@ -4,9 +4,51 @@
...
+        <a href="#"><h3>Home</h3></a>
+        <a href="#"><h3>Login</h3></a>
+      </div>
+    </nav>
+
+    <div class="container-main flex-r">
+      <main class="main-content flex-c">
+        <h1>Quizily</h1>
+        <p>
+          Want to test your knowledge of some popular series, play our quiz!
```
Comment (mnath30) [inline index.html:26]: Change the style of this text since it is not visible.

**Review by mnath30 — APPROVED** [review]
Looks good to me.

File: index.html (hunk 4 above)
Comment (AmishMishra11) [reply index.html:26]: ok i will do that

**Pull request merged**